install.sh

```diff
@@ -312,6 +324,64 @@ mkdir -p "$DIR"
 cd "$DIR"
 
 # ========================================================================================
+# Sandcats.io - Dynamic DNS service hosted by Sandstorm.io
+if [ "yes" = "$USE_SANDCATS" ] ; then
+  # We generate the public key before prompting for a desired hostname
+  # so that when the user presses enter, we can try to register the
+  # hostname, and if that succeeds, we are totally done. This avoids a
+  # possible time-of-check-time-of-use race.
+  echo "As a Sandstorm user, you are invited to use a free Internet hostname as a subdomain of sandcats.io."
+
+  # The sandcats service places its authentication files in $DIR/sandcats.
```

Comment:
Perhaps it should be `$DIR/var/sandcats`? I like consolidating user data under `var`, while everything else is stuff that can be re-downloaded.